sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -16,39 +19,129 @@ to install.
 SYCL support: follow the instructions under
 [SYCL* Compiler and Runtimes](https://github.com/intel/llvm/releases/tag/2019-07)
 
+### Get the required tools:
+
+   a. `git` - for downloading the sources
+   
+   b. `cmake` - for building the compiler and tools (Get it at: http://www.cmake.org/download)
+      
+   c. `python` - for building the compiler and running tests (Get it at: https://www.python.org/downloads/release/python-2716/ )
+
+   d. `Visual Studio 2017 or later` (Windows only)
+
+
+---
+---
+---
+
 # Build the SYCL compiler and runtime
 
-Download the LLVM* repository with SYCL support to your local machine folder
-e.g. `$HOME/sycl` (assuming environment var `$SYCL_HOME`) folder using
-following command:
+Download the LLVM repository with SYCL support and required tools to your local machine folder, e.g. $WORK_DIR. 
+For simplicity it is assumed below that environment variable WORK_DIR contains path to an existing folder.
+
+## Set some `environment variables`. For example:
```

Comment:
```suggestion
## Configure environment
```

Reply:
Ok, fixed. You'll see the change in the updated version of the commit.